dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level

```diff
@@ -39,5 +39,35 @@
         <block type="maze_nectar"/>
       </xml>
     </required_blocks>
+    <solution_blocks>
+      <xml>
+        <block type="when_run" movable="false">
+          <next>
+            <block type="maze_moveForward">
+              <next>
+                <block type="maze_moveForward">
+                  <next>
+                    <block type="maze_turn">
+                      <title name="DIR">turnLeft</title>
+                      <next>
+                        <block type="maze_moveForward">
```

Comment:
that's correct! The idea is that this is the stuff that should be displayed when the teacher selects "View Solution" for a level